JPS_Ontology/ontology/ontosensorloggermobileapp/OntoSensorLoggerMobileApp.csv

```diff
@@ -0,0 +1,25 @@
+hasXComponent,Object Property,,,GravityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the acceleration vector due to gravity with respect to device's coordinate reference system in the x-direction (Short edge).,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasXComponent
+hasYComponent,Object Property,,,GravityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the acceleration vector due to gravity with respect to device's coordinate reference system in the y-direction (Long edge).,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasYComponent
+hasZComponent,Object Property,,,GravityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the acceleration vector due to gravity with respect to device's coordinate reference system in the z-direction (Through the phone).,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasZComponent
+hasXComponent,Object Property,,,MagneticFluxDensityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,only,A relationship that defines the direction of magnetic north with respect to the device in the x direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasXComponent
+hasYComponent,Object Property,,,MagneticFluxDensityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,only,A relationship that defines the direction of magnetic north with respect to the device in the y direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasYComponent
+hasZComponent,Object Property,,,MagneticFluxDensityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,only,A relationship that defines the direction of magnetic north with respect to the device in the z direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasZComponent
+https://www.theworldavatar.com/kg/ontodevice/measures,Object Property,,IS-A,https://www.theworldavatar.com/kg/ontodevice/Magnetometer,MagneticFluxDensityVector,,,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,measures
+https://www.theworldavatar.com/kg/ontodevice/measures,Object Property,,IS-A,https://www.theworldavatar.com/kg/ontodevice/Accelerometer,AccelerationVector,,,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,measures
+https://www.theworldavatar.com/kg/ontodevice/measures,Object Property,,IS-A,https://www.theworldavatar.com/kg/ontodevice/GravitySensor,GravityVector,,,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,measures
```

Comment:
You're providing three separate definitions of `measures`, each with different domain and range. This means these domain and range definitions will all appear alongside each other in the OWL, which will be interpreted as _intersection_. This does not make sense. I believe the correct way of defining this is the following:
Define `measures` with the UNION of `Magnetometer`, `Accelerometer`, and `GravitySensor` as domain, and the analogous union of vectors as range.
Then define restrictions of `measures` using the `only` quantifier for each domain/range pair, i.e. Magnetometer/MagneticFluxDensityVector, Accelerometer/AccelerationVector, GravitySensor/GravityVector.
This also implies that it is probably best to move those definitions to OntoDevice (because the restrictions will affect the class definitions). (If that means that _everything_ ends up in OntoDevice, and there is nothing left for OntoSensorLoggerMobileApp, then that's fine - I don't see a problem with that.)

Reply:
Hi Seb, I have spoke to Wilson @Abyssus1010 . We thought it would be the best to shift the definitions for the vector classes and the ObjectProperty "hasA" to OntoDevice. 

Since everything can be defined in OntoDevice. At this point of time, there is no specific use cases for OntoSensorLoggerMobileApp, therefore I think it would be best to close this branch/ontology for the time being.

Reply:
> We thought it would be the best to shift the definitions for the vector classes and the ObjectProperty "hasA" to OntoDevice. Since everything can be defined in OntoDevice. At this point of time, there is no specific use cases for OntoSensorLoggerMobileApp

I'm happy with that. However, rather than simply deleting this branch (and thus throwing away the revision history), I would suggest to continue on this branch, and do all the necessary changes here. You can edit the title of this pull request.